l10n_ro_account_edi_ubl/models/account_edi_xml_cius_ro.py

```diff
@@ -174,6 +174,26 @@ def _get_invoice_payment_means_vals_list(self, invoice):
                 )
         return res
 
+    def _import_fill_invoice_line_form(
+        self, journal, tree, invoice, invoice_line, qty_factor
+    ):
+        res = super(AccountEdiXmlCIUSRO, self)._import_fill_invoice_line_form(
+            journal, tree, invoice, invoice_line, qty_factor
+        )
+        tax_nodes = tree.findall(".//{*}Item/{*}ClassifiedTaxCategory/{*}ID")
+        if len(tax_nodes) == 1:
+            if tax_nodes[0].text in ["O", "E", "Z"]:
+                tax = self.env["account.tax"].search(
+                    [
+                        ("amount", "=", "0"),
+                        ("type_tax_use", "=", journal.type),
+                        ("amount_type", "=", "percent"),
```

Comment:
de adaugat  compania

Reply:
Nu se pot mapa cumva taxele din xml cu cele din Odoo?

Reply:
@dhongu add filtru dupa companie.
@danila12 nu se pot mapa, in xml vine doar cota TVA (%)